cmd/sonobuoy/app/images_test.go

```diff
@@ -99,9 +99,9 @@ func TestGetClusterVersion(t *testing.T) {
 			output, err := getClusterVersion(tc.input, Kubeconfig{})
 			switch {
 			case err != nil && !tc.expectErr:
-				t.Fatalf("Expected no error but got %v", err)
+				t.Errorf("Expected no error but got %v", err)
```

Comment:
When developing here I found I needed the next messages and not just a fatal here.